WrightTools/kit/_leastsq.py

```diff
@@ -1,4 +1,4 @@
-"""Interpolation tools."""
+"""Least-square fitting tools."""
```

Comment:
Any particular reason to add this to this PR?

It's a small enough change that now that it's done, not worth reverting, but it is unrelated.